### PR TITLE
part2_test

### PR DESCRIPTION
**Pull request by mrdannydanny**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,16 +2,58 @@
...
 ```
 export AWS_ACCESS_KEY_ID="<your-access-id>"           
 export AWS_SECRET_ACCESS_KEY="<your-secret-key>"
+export AWS_REGION='us-west-2'
+```
+
+3. Export the variables needed for the terraform code using TF_VAR_<variable_name> 
+
+```
+export TF_VAR_aws_access_key="<your_aws_access_key>"
```
Comment (diegows) [inline README.md]: Actually, using AWS_* standard env vars should be enough.

File: README.md [hunk 2]
```diff
@@ -2,16 +2,58 @@
...
+1. cd packer
+
+2. /usr/local/bin/packer validate example.json
+
+3. /usr/local/bin/packer build example.json
+
+note: the image name was found by using the aws cli describe command:
+`ws ec2 describe-images --owners self amazon --region us-west-2`
+
+4. copy the ami generated by packer, we'll use it to launch our ec2 in the next step.
```
Comment (diegows) [inline README.md]: What do we need to copy? How?

File: ec2_instance_terraform/base.tf [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      "Effect": "Allow",
+      "Resource": "*"
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_instance" "dannyinstance_1" {
+  ami           = var.aws_ami
```
Comment (diegows) [inline ec2_instance_terraform/base.tf]: Can we use "data sources" to avoid managing ami IDs?

**Review by diegows — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (danzarov) [reply README.md]: I'll fix it so we can use data sources instead of copying the ami ID manually :)

File: ec2_instance_terraform/base.tf (hunk 3 above)
Comment (danzarov) [reply ec2_instance_terraform/base.tf]: Sure! thanks for the advice

**Review by diegows — APPROVED** [review]

**Pull request merged**